app/controllers/CreditsSummaryController.scala

```diff
@@ -0,0 +1,150 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import auth.MtdItUser
+import config.featureswitch.{FeatureSwitching, MFACreditsAndDebits}
+import config.{AgentItvcErrorHandler, FrontendAppConfig, ItvcErrorHandler}
+import controllers.agent.predicates.ClientConfirmedController
+import controllers.predicates._
+import models.financialDetails.{DocumentDetail, FinancialDetailsErrorModel, FinancialDetailsModel}
+import play.api.Logger
+import play.api.i18n.I18nSupport
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents, Result}
+import services.{FinancialDetailsService, IncomeSourceDetailsService}
+import uk.gov.hmrc.auth.core.AuthorisedFunctions
+import uk.gov.hmrc.http.HeaderCarrier
+import views.html.CreditsSummary
+
+import java.net.URI
+import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
+
+class CreditsSummaryController @Inject()(creditsView: CreditsSummary,
+                                         val authorisedFunctions: AuthorisedFunctions,
+                                         incomeSourceDetailsService: IncomeSourceDetailsService,
+                                         financialDetailsService: FinancialDetailsService,
+                                         itvcErrorHandler: ItvcErrorHandler,
+                                         checkSessionTimeout: SessionTimeoutPredicate,
+                                         retrieveBtaNavBar: NavBarPredicate,
+                                         authenticate: AuthenticationPredicate,
+                                         retrieveNino: NinoPredicate,
+                                         retrieveIncomeSources: IncomeSourceDetailsPredicate)
+                                        (implicit val appConfig: FrontendAppConfig,
+                                         mcc: MessagesControllerComponents,
+                                         val ec: ExecutionContext,
+                                         val agentItvcErrorHandler: AgentItvcErrorHandler
+                                        ) extends ClientConfirmedController with FeatureSwitching with I18nSupport {
+
+  private def getFinancialsByTaxYear(calendarYear: Int, isAgent: Boolean)(f: List[DocumentDetail] => Future[Result])
+                                    (implicit user: MtdItUser[AnyContent]): Future[Result] = {
+    financialDetailsService.getFinancialDetails(calendarYear, user.nino).flatMap {
+      case FinancialDetailsModel(_, _, documentDetails, _) =>
+        val docDetailsMFACredits = documentDetails.filter(_.validMFACreditDescription())
+
+        val financialDetailsWithCredit: List[DocumentDetail] = docDetailsMFACredits.filter { dd =>
```

Comment:
I guess we don't need audit events for this page??

Reply:
Yes, I think we do not need the audit events here